.github/workflows/reusable-docs.yml

```
@@ -40,6 +40,7 @@ jobs:
       uses: Ana06/get-changed-files@v2.2.0
       with:
         filter: "Doc/**"
+        format: csv  # unbreaks paths /w whitespaces; json is harder to parse
```

But this is used as input for the touch command:

```yml
touch ${{ steps.changed_files.outputs.added_modified }}
```

And this wasn't tested in the demo PR: https://github.com/python/cpython/actions/runs/5135145133/jobs/9240173418?pr=105153

A quick local test: `touch tox.ini,requirements.txt` creates a single file called `tox.ini,requirements.txt`, it doesn't touch the two files.

We'll need some handling to parse the comma-separated output into something that touches those files.

Note, a bit later, we have another file `Doc/tools/touch-clean-files.py` that reads the contents of `Doc/tools/.nitignore` and touches the files listed there. Perhaps we could modify that script to take an optional input argument, the CSV string? When present, touch those instead of `.nitignore`.

---

```suggestion
        format: csv  # works for paths with spaces
```

---

Sounds good. Though, I can't make the changes right now — got a fever and will probably recover in a few days..

---

Oh, take care and get well soon! Absolutely no rush on this.

---

https://github.com/webknjaz/cpython/pull/168 is a PR to update this one and add support for CSV input. It allows `Doc/tools/touch-clean-files.py` to take a `--clean` parameter, which is a comma-separated list of files to touch:

This means we can use https://github.com/Ana06/get-changed-files with paths containing spaces.

Demo PR:

* https://github.com/hugovk/cpython/pull/50

With added commit with two changed docs files, one with a space in the path:

* https://github.com/hugovk/cpython/pull/50/commits/de2d60ebb52859b9af10ed91beb69db48b8f3a13

First action check:

* https://github.com/hugovk/cpython/actions/runs/5141469456/jobs/9253999311?pr=50#step:5:18

Second action check:

* https://github.com/hugovk/cpython/actions/runs/5141469456/jobs/9254006575?pr=50#step:8:22

Second action output is used here, the two files are touched:

```
Touching:
Doc/about.rst
Doc/tools/touch-clean-files.py
Touched 2 files
```

* https://github.com/hugovk/cpython/actions/runs/5141469456/jobs/9254006575?pr=50#step:9:16

---

Thanks! Absorbing your commits now...